lib/msal-common/src/client/AuthorizationCodeClient.ts

```diff
@@ -413,20 +413,18 @@ export class AuthorizationCodeClient extends BaseClient {
             );
         }
 
-        if (request.tokenBodyParameters) {
+        if (request.extraParams) {
             RequestParameterBuilder.addExtraQueryParameters(
```

Comment:
Usage of `addExtraQueryParamters` is confusing here since we're not putting it in the QS. Let's consider renaming this to `addExtraParameters` which we can use for both sets of params. Since the placement in QS/body happens outside of this I don't think the parameter builder needs to be "aware" of QS vs body

Reply:
Consolidated both addExtraQS and addPostBodyParams into addExtraParameters, thanks!